Assets/MRTK/Core/Definitions/SpatialAwareness/SpatialAwarenessSurfaceTypes.cs

```diff
@@ -53,16 +51,16 @@ public enum SpatialAwarenessSurfaceTypes
         /// <para>Background should not be confused with Unknown. There is sufficient data to 
         /// classify the surface and it has been found to not correspond to a defined type.</para>
         /// </remarks>
-        Background = 1 << 29,
+        Background = 1 << 5,
```

Comment:
> This PR fixes the issue of the set surface types in the SU profile does not correctly translate into matching behavior due to the way the enum was setup.

I know we talked about this offline, but, for future reference for others, could you expand a bit here on what the underlying issue was please?

Reply:
Sure! In order to allow users to change the observed surface type by the observer, we create a property field for the serialized property storing the enum flags in the profile editor code. The problem is Unity's default property field assumes the enum is represented by consecutive powers of two which was not the case before the change. This mismatch led to unexpected behavior as the user's input to the inspector of the profile cannot be mapped to the correct enum flags.